framework/util/options.cpp

```diff
@@ -71,77 +73,81 @@ std::vector<FrameRange> GetFrameRanges(const std::string& args)
             }
             else
             {
-                GFXRECON_LOG_WARNING(
-                    "Ignoring invalid screenshot frame range \"%s\", which contains non-numeric values", range.c_str());
+                GFXRECON_LOG_WARNING("Ignoring invalid range \"%s\" for %s, which contains non-numeric values",
+                                     range.c_str(),
+                                     option_name);
                 invalid = true;
                 break;
             }
         }
 
         if (!invalid)
         {
-            FrameRange screenshot_range;
+            UintRange uint_range;
```

Comment:
Could this be `boundary_range` or `trim_range` or something similar that is more descriptive?

Reply:
The intent is for this to be a generic function to parse uint ranges and the current name reflects that. Besides trim frame and queue submit ranges it is also used for screenshot ranges and could be used elsewhere range options are parsed (e.g., [`GetMeasurementFrameRange`](https://github.com/LunarG/gfxreconstruct/blob/016ac347a71c62e848715d47fa0a71629784a1ff/tools/tool_settings.h#L501))

Reply:
Okay, yeah, that makes sense!